semcore/color-picker/src/components/colors.tsx

```diff
@@ -33,18 +38,26 @@ export function Colors(props: ColorsAsProps) {
 export function ColorsCustom(props: ColorsCustomAsProps) {
   const { Children, styles, colors, onPlusButtonClick, getI18nText } = props;
   const SColors = Root;
+  const SColorsContainer = Flex;
   const SPlusButton = 'div';
 
   return sstyled(styles)(
-    <SColors render={Box} role='list' aria-label={getI18nText('customColors')}>
-      {Children.origin ? (
-        <Children />
-      ) : (
-        colors?.map((color) => <PaletteManager.Item value={color} key={color} />)
-      )}
-      <SPlusButton onClick={onPlusButtonClick} aria-hidden='true'>
+    <SColorsContainer>
+      <SColors
+        render={Box}
+        role='listbox'
+        aria-orientation='horizontal'
+        aria-label={getI18nText('customColors')}
+      >
+        {Children.origin ? (
+          <Children />
+        ) : (
+          colors?.map((color) => <PaletteManager.Item value={color} key={color} />)
+        )}
+      </SColors>
+      <SPlusButton onClick={onPlusButtonClick} role='button' aria-label={getI18nText('addColor')}>
         <MathPlusM color='gray-500' />
```

Comment:
Let's update the color to `icon-primary-neutral`, as in Figma 🙏

Reply:
fixed